### PR TITLE
Added Works section and About myself

### PR DESCRIPTION
**Pull request by ameerhamzakhalid**
I added works section module in html.
I added images required for works section layout according to template.
I added CSS properties required for works section module.
I added About myself module in html.
I added images required for About myself section layout according to template.
I added CSS properties required for About myself section module.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,5 +95,179 @@ <h2>
         </button>
     </div>
   </section>
```
Comment (topeogunleye) [inline index.html:98]: - [x] [REQUIRED] As the previous code Reviewer suggested. All of your project sections should be within a HTML main tag and give it the main container a `display: grid`

EXAMPLE

`<main class="main-container">`
    `<section>`
     ...
    `</section>`


`</main>`

.main-container {
 display: grid;
}

**Review by topeogunleye — CHANGES_REQUESTED** [review]
## Hi @ameerhamzakhalid,

Good job so far!

## Highlights!
- Good branch name ✔️ 
- No linter errors ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝. However, there are some issues that you still need to work on to go to the next project but you are almost there!
I noticed you made a new pull request after your previous one was invalidated. Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@topeogunleye` if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ameerhamzakhalid) [reply index.html:98]: Thank you for your kind response. display: grid; is already added in CSS file under work container on line 253

Comment (ameerhamzakhalid) [reply index.html:98]: asdf
You can see that the first reviewer admitted that I already added  display: grid;

Comment (topeogunleye) [reply index.html:98]: Please Kindly surround the main sections of the page with a main tag, then submit for a rereview. This is better for [semantics](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main). The  main container should also have a display grid
I'll check and get back to you pls

Edit: Please just change the div tag you used on line 99 in index.html to a main tag for sematics.

**Comment by ameerhamzakhalid** [comment]
@topeogunleye  kindly check now I added the main Section of the page and added display grid in it.

**Comment by topeogunleye** [comment]
> @topeogunleye kindly check now I added the main Section of the page and added display grid in it.

Excellent. Please submit for a rereview. All changes are completed.

**Review by smunozmo — APPROVED** [review]
Hi @ameerhamzakhalid, 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 👏 👏 👏 




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**